db/data/20190917122531_populate_student_loans_employments.rb

```diff
@@ -0,0 +1,24 @@
+class PopulateStudentLoansEmployment < ActiveRecord::Migration[5.2]
+  def up
+    StudentLoans::Eligibility.where.not(claim_school: nil).where(employments: nil).each do |eligibility|
+      employment = StudentLoans::Employment.create!(
+        school: eligibility.selected_employment.school,
+        biology_taught: eligibility.biology_taught,
+        chemistry_taught: eligibility.chemistry_taught,
+        computer_science_taught: eligibility.computer_science_taught,
+        languages_taught: eligibility.languages_taught,
+        physics_taught: eligibility.physics_taught,
+        taught_eligible_subjects: eligibility.taught_eligible_subjects,
+        student_loan_repayment_amount: eligibility.student_loan_repayment_amount,
+      )
+
+      eligibility.update!(
+        employments: [employment]
+      )
```

Comment:
I don't think this migration will actually work, now. `StudentLoans::Employment.create!` needs a reference to `eligibility`. Let's change this to:

```ruby
eligibility.update!(employments: [StudentLoans::Employment.build(...)])
```

Reply:
It seems that this data migration is of sufficient complexity that would benefit from a unit test (either by running the migration directly from a spec, if that's possible, or by extracting the `up` logic to something we _can_ test). 

Probably not something that I would have commented upon otherwise, but I was swung by the fact that we're speculating about whether it's going to work.

Reply:
Agreed

Reply:
I'm sure we can require the migration and run the up on it manually to test it. This seems like a useful thing to work out how to do.

Reply:
Probably also worth running `down` and seeing that it goes back to the initial state.

Reply:
I've added a unit test for the data migration, we'll have to remove/modify it when we end up removing the columns we're migrating from